main/auth/auth.py

```diff
@@ -57,7 +57,7 @@ def key(self):
     return self.user_db.key.urlsafe()
 
   def get_id(self):
-    return self.user_db.key.urlsafe()
+    return unicode(self.user_db.session_token)
```

Comment:
Possibly, I was just filling the [docs](https://flask-login.readthedocs.io/en/latest/#your-user-class):
>get_id()
This method must return a unicode that uniquely identifies this user, and can be used to load the user from the user_loader callback. Note that this must be a unicode - if the ID is natively an int or some other type, you will need to convert it to unicode.